index.html

```diff
@@ -0,0 +1,444 @@
+                    </div>
+                    <p class="team-card__title">
+                        CO Founder
+                    </p>
+                    <div class="social social--spacing-md">
+                        <img class="social__link" src="./assets/svgs/fbvector.svg" alt="facebook">
+                        <img class="social__link" src="./assets/svgs/instavesctor.svg" alt="instagram">
+                        <img class="social__link" src="./assets/svgs/twitterVector.svg" alt="twitter">
+                    </div>
+                </div>
+            </div>
+        </div>
+    
+        <div class="contact-us contact-us--spacing-midium ">
+            <div class="container container--spacing-small">
+                <div class="contact-us__title">
+                    Contact Us
+                </div>
+                <h1 class="site-header site-header--spacing-sm" >
+                    Make an Appointment
+                </h1>
+            </div>
+            <div class="container container--spacing-small">
+                <form accept="#" class="contact-us__form">
+                    <input class="contact-us__field" type="text" placeholder="Fullname">
+                    <input class="contact-us__field" type="email" placeholder="example@email.com">
+                    <select class="contact-us__field" id="">
+                        <option value="">Please selct</option>
+                    </select>
+                    <select class="contact-us__field" id="">
+                        <option value="">04:00 Available</option>
+                    </select>
+                    <textarea class="contact-us__field" name="" id="" cols="30" rows="10" placeholder="Message"></textarea>
+                    <button class="button button--primary button--rounded button--size-small" >
+                        Book Appointment
+                    </button>
+                </form>
+            </div>
+        </div>
+
+        <div class="footer footer--dark">
+            <div class="container">
+                <div class="footer__nav">
+                    <h1 class="logo logo--white">KMarketer</h1>
+                    <div class="social">
+                        <img class="social__link" src="./assets/svgs/fb.svg" alt="facebook">
+                        <img class="social__link" src="./assets/svgs/instagram.svg" alt="facebook">
+                        <img class="social__link" src="./assets/svgs/twitter.svg" alt="facebook">
+                        <img class="social__link" src="./assets/svgs/youtube.svg" alt="facebook">
+                    </div>
+                </div>
+                <div class="footer__line"></div>
+                <div class="footer__links">
+                    <div class="footer__list">
+                        <div class="footer__title">
+                            Company Info
+                        </div>
+                        <ul class="navigation navigation--col navigation--right">
```

Comment:
reference `navigation` to its block `footer`

Reply:
Navigation is A block and this was the second time I used it, I changed the name if you considered navigation as the header navigation it just lists of element which are ordered in different way